components/raftstore/src/coprocessor/config.rs

```diff
@@ -5,6 +5,9 @@ use crate::store::SplitCheckTask;
 
 use configuration::{ConfigChange, ConfigManager, Configuration};
 use engine_traits::{config as engine_config, PerfLevel};
+use serde::*;
```

Comment:
The reason we import macros with the `use` syntax is that we want to make it easy to find the definition and source of imported macros. IMO, using `::*` will go against the original intention why we want to clean up the `extern crate xxx`.